[2024-01-24T03:23:18.176+0800] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: youtube_comments_score.train_model scheduled__2024-01-23T18:05:00+00:00 [queued]>
[2024-01-24T03:23:18.189+0800] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: youtube_comments_score.train_model scheduled__2024-01-23T18:05:00+00:00 [queued]>
[2024-01-24T03:23:18.189+0800] {taskinstance.py:1288} INFO - 
--------------------------------------------------------------------------------
[2024-01-24T03:23:18.189+0800] {taskinstance.py:1289} INFO - Starting attempt 2 of 2
[2024-01-24T03:23:18.189+0800] {taskinstance.py:1290} INFO - 
--------------------------------------------------------------------------------
[2024-01-24T03:23:18.207+0800] {taskinstance.py:1309} INFO - Executing <Task(BashOperator): train_model> on 2024-01-23 18:05:00+00:00
[2024-01-24T03:23:18.214+0800] {standard_task_runner.py:55} INFO - Started process 110117 to run task
[2024-01-24T03:23:18.227+0800] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'youtube_comments_score', 'train_model', 'scheduled__2024-01-23T18:05:00+00:00', '--job-id', '46', '--raw', '--subdir', 'DAGS_FOLDER/youtube_comments_score.py', '--cfg-path', '/tmp/tmpq27xhoy0']
[2024-01-24T03:23:18.229+0800] {standard_task_runner.py:83} INFO - Job 46: Subtask train_model
[2024-01-24T03:23:18.444+0800] {task_command.py:389} INFO - Running <TaskInstance: youtube_comments_score.train_model scheduled__2024-01-23T18:05:00+00:00 [running]> on host ml
[2024-01-24T03:23:18.616+0800] {taskinstance.py:1516} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=admin
AIRFLOW_CTX_DAG_ID=youtube_comments_score
AIRFLOW_CTX_TASK_ID=train_model
AIRFLOW_CTX_EXECUTION_DATE=2024-01-23T18:05:00+00:00
AIRFLOW_CTX_TRY_NUMBER=2
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-01-23T18:05:00+00:00
[2024-01-24T03:23:18.618+0800] {subprocess.py:63} INFO - Tmp dir root location: 
 /tmp
[2024-01-24T03:23:18.619+0800] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'python3 /home/am/project/scripts/train_model.py']
[2024-01-24T03:23:18.640+0800] {subprocess.py:86} INFO - Output:
[2024-01-24T03:23:19.654+0800] {subprocess.py:93} INFO - /home/am/project/scripts/train_model.py:3: DeprecationWarning:
[2024-01-24T03:23:19.654+0800] {subprocess.py:93} INFO - Pyarrow will become a required dependency of pandas in the next major release of pandas (pandas 3.0),
[2024-01-24T03:23:19.654+0800] {subprocess.py:93} INFO - (to allow more performant data types, such as the Arrow string type, and better interoperability with other libraries)
[2024-01-24T03:23:19.654+0800] {subprocess.py:93} INFO - but was not found to be installed on your system.
[2024-01-24T03:23:19.654+0800] {subprocess.py:93} INFO - If this would cause problems for you,
[2024-01-24T03:23:19.654+0800] {subprocess.py:93} INFO - please provide us feedback at https://github.com/pandas-dev/pandas/issues/54466
[2024-01-24T03:23:19.655+0800] {subprocess.py:93} INFO - 
[2024-01-24T03:23:19.655+0800] {subprocess.py:93} INFO -   import pandas as pd
[2024-01-24T03:23:19.658+0800] {subprocess.py:93} INFO - Traceback (most recent call last):
[2024-01-24T03:23:19.659+0800] {subprocess.py:93} INFO -   File "/home/am/project/scripts/train_model.py", line 11, in <module>
[2024-01-24T03:23:19.659+0800] {subprocess.py:93} INFO -     with open('/home/am/project/models/data.pickle', 'wb') as f:
[2024-01-24T03:23:19.659+0800] {subprocess.py:93} INFO - FileNotFoundError: [Errno 2] No such file or directory: '/home/am/project/models/data.pickle'
[2024-01-24T03:23:19.881+0800] {subprocess.py:97} INFO - Command exited with return code 1
[2024-01-24T03:23:19.911+0800] {taskinstance.py:1776} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/am/project/venv/lib/python3.10/site-packages/airflow/operators/bash.py", line 196, in execute
    raise AirflowException(
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 1.
[2024-01-24T03:23:19.915+0800] {taskinstance.py:1327} INFO - Marking task as FAILED. dag_id=youtube_comments_score, task_id=train_model, execution_date=20240123T180500, start_date=20240123T192318, end_date=20240123T192319
[2024-01-24T03:23:19.941+0800] {standard_task_runner.py:100} ERROR - Failed to execute job 46 for task train_model (Bash command failed. The command returned a non-zero exit code 1.; 110117)
[2024-01-24T03:23:19.987+0800] {local_task_job.py:212} INFO - Task exited with return code 1
[2024-01-24T03:23:20.036+0800] {taskinstance.py:2596} INFO - 0 downstream tasks scheduled from follow-on schedule check
